mlir/test/Dialect/Rock/test_vectorization_inference.mlir

```diff
@@ -109,24 +173,24 @@ func.func @test_vectorization() {
   // CHECK-NEXT: result = 8
   %15 = "get_length"() {transforms = [#transform_map1, #transform_map0, #transform_map3, #transform_map4],
     in_dim = 0 : index, max_len = 8 : index} : () -> (memref<8xf32>)
-
+ 
   // Padding by vector lengths behaves
   // CHECK-NEXT: result = 4
   %16 = "get_length"() {transforms = [#transform_map5], in_dim = 0 : index, max_len = 8 : index} : () -> (memref<4xf32>)
   // Left and right padding apply separately
   // CHECK-NEXT: result = 2
   %17 = "get_length"() {transforms = [#transform_map6], in_dim = 0 : index, max_len = 8 : index} : () -> (memref<4xf32>)
-
+ 
   // Intervening dimensions prevent merge
   // CHECK-NEXT: result = 8
   %18 = "get_length"() {transforms = [#transform_map7], in_dim = 0 : index, max_len = 32 : index} : () -> (memref<4x5x8xf32>)
   // Unless it was an illusion
-  // CHECK-NEXT: result = 32
+  //  CHECK-NEXT: result = 32
   %19 = "get_length"() {transforms = [#transform_map7, #transform_map8], in_dim = 0 : index, max_len = 32 : index} : () -> (memref<4x8xf32>)
   // or a broadcast
   // CHECK-NEXT: result = 32
   %20 = "get_length"() {transforms = [#transform_map7, #transform_map9], in_dim = 0 : index, max_len = 32 : index} : () -> (memref<4x1x8xf32>)
-
+ 
```

Comment:
Nit: you added/removed a bunch of spaces, can you undo that? (The editor setting that automatically strips trailing spaces from lines is usually somewhere and may be a good call)